hudi-cli/src/main/java/org/apache/hudi/cli/commands/SparkMain.java

```diff
@@ -458,7 +458,7 @@ protected static int upgradeOrDowngradeTable(JavaSparkContext jsc, String basePa
             .setLoadActiveTimelineOnLoad(false).setConsistencyGuardConfig(config.getConsistencyGuardConfig())
             .setLayoutVersion(Option.of(new TimelineLayoutVersion(config.getTimelineLayoutVersion()))).build();
     try {
-      new UpgradeDowngrade(metaClient, config, new HoodieSparkEngineContext(jsc), SparkUpgradeDowngradeHelper.getInstance())
+      new Versioner(metaClient, config, new HoodieSparkEngineContext(jsc), SparkUpgradeDowngradeHelper.getInstance())
```

Comment:
Not a strong opinion.  I see you renamed upgrade- and downgrade-related classes. This isn't in the scope of the class renaming originally.  If we agree that the new names are better,

Reply:
Reverted